CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
 
+## 0.16.0 - 2020-12-04
+### Added
+- MongoDB server defaults to version “3.6”
+
+### Changed
+- Replaced deprecated azure sdk with new sdk:
+  - “documentdb” to “@azure/cosmos”
+  - “azure-arm-cosmosdb” to “@azure/arm-cosmosdb”
+  - “azure-arm-postgresql” to “@azure/arm-postgresql”
+  - Upgraded following packages:
+    - mongodb: "3.3" to "3.6"
+    - “vscode-languageclient" and “vscode-languageserver" to latest versions
+
```

Comment:
Do we need to mention all of these package updates here?

Reply:
No we don't mention engineering work like this unless it's in the context of a feature/enhancement. For example, like this:
> Improved extension activation time (by switching to the [azure-sdk-for-js](https://github.com/Azure/azure-sdk-for-js))

Reply:
That's fine if it means the changelog doesn't have much in it, but I might do 0.15.1 instead of 0.16.0 in that case

Reply:
To be clear, you can add that exact line I mentioned because your work to update the packages did help with activation time. We put that line in other changelogs when we updates those packages for other extensions